src/box/iproto.cc

```diff
@@ -3291,6 +3425,81 @@ iproto_thread_init(struct iproto_thread *iproto_thread)
 	iproto_thread->requests_in_stream_queue = 0;
 }
 
+/**
+ * Returns `enum iproto_type' if `name' is a valid name of the IPROTO overriding
+ * event, or iproto_type_MAX otherwise. If the name contains correct prefix, but
+ * the request type is invalid, the error is logged with CRIT log level.
+ * `is_by_id' set to True if the request is overridden by id, False if by name.
+ */
+static uint32_t
+get_iproto_type_from_event_name(const char *name, bool *is_by_id)
+{
+	*is_by_id = false;
+	const char *prefix = "box.iproto.override";
+	const size_t prefix_len = strlen(prefix);
+	if (strncmp(name, prefix, prefix_len) != 0)
+		return iproto_type_MAX;
+
+	const char *req_name = name + prefix_len;
+	uint32_t type = iproto_type_MAX;
+
+	if (*req_name == '[') {
+		char *end;
+		req_name++;
+		*is_by_id = true;
+		type = strtol(req_name, &end, 10);
+		if (req_name == end)
+			return iproto_type_MAX;
```

Comment:
`box.iproto.override[select]` or `box.iproto.override[15].header` are invalid names which could very possibly come here - they should be logged too.
More strictly, let's log change on every event, that belongs to `box.iproto.override` namespace but does not override any request handler.

Reply:
Done.